entrypoint.sh

```diff
@@ -17,14 +30,19 @@ else
   VERSION='patch';
 fi
 
+# What is this? 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
Its the email of github-actions bot
Providing this email authenticates the bot as submitter, and shows the avatar of the bot in commits